src/app/+player/player.component.html

```diff
@@ -1,39 +1,59 @@
-<metadata>
-  <artwork [style.background-image]="artworkUrl"></artwork>
-  <section>
-    <header>
-      <h1 [title]="title">{{title}}</h1>
-      <h2 [title]="subtitle">{{subtitle}}</h2>
-    </header>
-    <nav>
-      <a (click)="showShareModal()">Share</a>
-      <a [href]="subscribeUrl" *ngIf="subscribeUrl" [target]="subscribeTarget">Subscribe</a>
-    </nav>
-  </section>
-</metadata>
-<controls (window:keydown)="handleHotkey($event)">
-  <buttons>
-    <button (click)="seekBy(-10)" id="rewind-control-button">
-      <img src="/images/back10.svg" alt="">
-    </button>
-    <button (click)="togglePlayPause()" id="playback-control-button">
-      <img src="/images/play.svg" alt="" *ngIf="player.paused">
-      <img src="/images/pause.svg" alt="" *ngIf="!player.paused">
-    </button>
-    <button (click)="cyclePlaybackRate()" id="playback-rate-control-button">
-      <div *ngIf="player.playbackRate === 1">1<span>✕</span></div>
-      <div *ngIf="player.playbackRate === 1.25">1.25<span>✕</span></div>
-      <div *ngIf="player.playbackRate === 1.5">1.5<span>✕</span></div>
-      <div *ngIf="player.playbackRate === 2">2<span>✕</span></div>
-    </button>
-  </buttons>
-  <scrubber>
-    <progress-bar
-      [value]="currentTime | async"
-      [maximum]="duration | async"
-      (seek)="onSeek($event)"
-      (scrubbing)="onScrub($event)"></progress-bar>
-    <playback-progress>{{currentTime | async | duration}}</playback-progress>
-    <duration>{{duration | async | duration}}</duration>
-  </scrubber>
-</controls>
+
+<background-img [style.background-image]="artworkUrl"></background-img>
```

Comment:
as discussed!: ng did not like: `Template parse errors:
Only void and foreign elements can be self closed "background-img" ("[ERROR ->]<background-img [style.background-image]="artworkUrl" />`